src/AbstractBlocklockReceiver.sol

```diff
@@ -111,28 +111,6 @@ abstract contract AbstractBlocklockReceiver is IBlocklockReceiver, ConfirmedOwne
         return blocklock.pendingRequestExists(subId);
     }
 
```

Comment:
We can also removed `function getBalance() ` because it is also for checking balance for direct funding